From 03d92df03faa1c113d30374fdb220e0a33815347 Mon Sep 17 00:00:00 2001
From: Kihwal Lee <kihwal@apache.org>
Date: Wed, 13 Apr 2016 11:24:45 -0500
Subject: [PATCH 1797/2748] =?UTF-8?q?HDFS-10270.=20TestJMXGet:testNameNode()?=
 =?UTF-8?q?=20fails.=20Contributed=20by=20Gergely=20Nov=C3=A1k?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

(cherry picked from commit d2f3bbc29046435904ad9418073795439c71b441)
(cherry picked from commit ccaf746eeacfafc7392a994f28ee9dbf595c84af)
(cherry picked from commit 9f5a22ff22b4841f78ac87c71c291456d20ef4d6)

Change-Id: I67609bb74fff48fb92feb6b414fe6eb1f1453785
---
 .../java/org/apache/hadoop/tools/TestJMXGet.java   |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java
index d9a3b79..ce98cfd 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/tools/TestJMXGet.java
@@ -120,9 +120,6 @@ public void testNameNode() throws Exception {
         jmx.getValue("NumLiveDataNodes")));
     assertGauge("CorruptBlocks", Long.parseLong(jmx.getValue("CorruptBlocks")),
                 getMetrics("FSNamesystem"));
-    DFSTestUtil.waitForMetric(jmx, "NumOpenConnections", numDatanodes);
-    assertEquals(numDatanodes, Integer.parseInt(
-        jmx.getValue("NumOpenConnections")));
 
     cluster.shutdown();
     MBeanServerConnection mbsc = ManagementFactory.getPlatformMBeanServer();
-- 
1.7.9.5

